sql-cli/sql_cli/__main__.py

```diff
@@ -13,10 +13,10 @@
 from sql_cli.astro.command import AstroCommand
 from sql_cli.astro.group import AstroGroup
 from sql_cli.constants import DEFAULT_AIRFLOW_HOME, DEFAULT_DAGS_FOLDER
-from sql_cli.exceptions import ConnectionFailed, DagCycle, EmptyDag, SqlFilesDirectoryNotFound
+from sql_cli.exceptions import ConnectionFailed, DagCycle, EmptyDag, WorkflowFilesDirectoryNotFound
 
 if TYPE_CHECKING:
-    from sql_cli.project import Project
+    from sql_cli.project import Project  # pragma: no cover
```

Comment:
Add a ticket to check on whether we can cover this line and remove the no cover.

Reply:
Created ticket: https://github.com/astronomer/astro-sdk/issues/1283